<launch>
	<include file="$(find rob_auto_cal)/launch/camera1394_launch.launch" />
	<node pkg="rob_auto_cal" name="chess_features" type="chess_features" args="0.0255" />
	<include file="$(find for_andre)/launch/ur10.launch" />
	<node pkg="rob_auto_cal" name="trans_calc" type="trans_calc" output="screen"/>
</launch>
 